packages/leancode_hooks/lib/src/use_synced_text_editing_controller.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+
+TextEditingController useSyncedTextEditingController(
+  void Function(TextEditingValue value) onChanged, {
+  String? initialText,
+}) {
+  final controller = useTextEditingController(text: initialText);
+
+  void listener() => onChanged(controller.value);
```

Comment:
How about moving the `listener` declaration inside the `useEffect`? It will be unnecessarily re-created every time the hook is called (not sure if this gets optimized during build)

Reply:
thanks sir